pkg/operation/botanist/component/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -509,8 +528,12 @@ func (k *kubeControllerManager) computeCommand(port int32) []string {
 	command = append(command,
 		fmt.Sprintf("--cluster-cidr=%s", k.podNetwork.String()),
 		fmt.Sprintf("--cluster-name=%s", k.namespace),
-		fmt.Sprintf("--cluster-signing-cert-file=%s/%s", volumeMountPathCAClient, secrets.DataKeyCertificateCA),
-		fmt.Sprintf("--cluster-signing-key-file=%s/%s", volumeMountPathCAClient, secrets.DataKeyPrivateKeyCA),
+		fmt.Sprintf("--cluster-signing-kube-apiserver-client-cert-file=%s/%s", volumeMountPathCAClient, secrets.DataKeyCertificateCA),
+		fmt.Sprintf("--cluster-signing-kube-apiserver-client-key-file=%s/%s", volumeMountPathCAClient, secrets.DataKeyPrivateKeyCA),
+		fmt.Sprintf("--cluster-signing-kubelet-client-cert-file=%s/%s", volumeMountPathCAClient, secrets.DataKeyCertificateCA),
+		fmt.Sprintf("--cluster-signing-kubelet-client-key-file=%s/%s", volumeMountPathCAClient, secrets.DataKeyPrivateKeyCA),
+		fmt.Sprintf("--cluster-signing-kubelet-serving-cert-file=%s/%s", volumeMountPathCAKubelet, secrets.DataKeyCertificateCA),
+		fmt.Sprintf("--cluster-signing-kubelet-serving-key-file=%s/%s", volumeMountPathCAKubelet, secrets.DataKeyPrivateKeyCA),
```

Comment:
Wouldn't  `--cluster-signing-key-file` also cover the certificate used for `--cluster-signing-legacy-unknown-cert-file` previously. IIUC It isn't necessary for k8s>=1.19 and when `certificates.k8s.io/v1` api is used, where it is no longer allowed to specify `kubernetes.io/legacy-unknwon` signerName, but some older functionality might depend on it (if someone still does not use the `certificates.k8s.io/v1` api)
So I think that `--cluster-signing-legacy-unknown-key-file` flag should also be added.

Reply:
OK, you are right, let's add it to be on the safe side. We can remove it when we drop support for older versions.